src/screens/ExplainSheet.js

```diff
@@ -106,13 +97,13 @@ export const explainers = network => ({
     emoji: '📊',
     extraHeight: -102,
     text: FLOOR_PRICE_EXPLAINER,
-    title: 'Collection floor price',
+    title: lang.t('explain.floor_price.title'),
   },
   gas: {
     emoji: '⛽️',
     extraHeight: 2,
     text: gasExplainer(network),
-    title: `${network} network fee`,
+    title: lang.t('explain.gas.title'),
```

Comment:
This is missing the `networkName` argument

Reply:
@jinchung Fixed now, thanks for catching that!